src/panel_gwalker/_gwalker.js

```diff
@@ -0,0 +1,69 @@
+import {GraphicWalker} from "graphic-walker"
+import {useEffect, useState, useRef} from "react"
+
+function transform(data) {
+  const keys = Object.keys(data);
+  const length = data[keys[0]].length;
+
+  return Array.from({ length }, (_, i) =>
+    keys.reduce((obj, key) => {
+      obj[key] = data[key][i];
+      return obj;
+    }, {})
+  );
+}
+
+function cleanToDict(value){
+    value = JSON.stringify(value)
+    value = JSON.parse(value)
+    return value
+}
+
+export function render({ model }) {
+  const [data] = model.useState('object')
+  const [fields] = model.useState('fields')
+  const [appearance] = model.useState('appearance')
+  const [config] = model.useState('config')
+  const [currentChart, setCurrentChart] = model.useState("current_chart")
+  const [saveCurrentChart,setSaveCurrentChart] = model.useState("save_current_chart")
+  const [currentChartList, setCurrentChartList] = model.useState("current_chart_list")
+  const [saveCurrentChartList,setSaveCurrentChartList] = model.useState("save_current_chart_list")
+  const [transformedData, setTransformedData] = useState([]);
+
+  const graphicWalkerRef = useRef(null);
+
+  if (saveCurrentChart && graphicWalkerRef && graphicWalkerRef.current){
+    graphicWalkerRef.current.exportChart().then((value)=>{
+        value=cleanToDict(value)
+        setCurrentChart(value)
+    })
+    setSaveCurrentChart(false)
```

Comment:
I had to set the save_current_chart event parameter to False manually in order to be able to click it again. Is this a bug?

Reply:
Not its called  `setExportCurrentChart(false)`. But the same issue.